cashctrl_api/client.py

```diff
@@ -256,6 +291,13 @@ def update_categories(self, resource: str, target: List[str],
                     params = {'name': nodes[i]}
                     if parent_path:
                         params['parentId'] = categories[parent_path]
+                    elif resource == 'account':
+                        parentId = categories[parent_path]
+                        if not parentId:
+                            raise ValueError(f"Parent for '${parent_path}' path doesn`t exist")
+                        params['parentId'] = parentId
```

Comment:
Would we loose anything by simplifying to:
```python
if parent_path:
    params['parentId'] = categories[parent_path]
elif resource == 'account':
    raise ValueError(f"Cannot create new root nodes for account categories: '{node_path}'.")
```
Instead of:
```python
if parent_path:
    params['parentId'] = categories[parent_path]
elif resource == 'account':
    parentId = categories[parent_path]
    if not parentId:
        raise ValueError(f"Parent for '${parent_path}' path doesn`t exist")
    params['parentId'] = parentId
```